source/docs/advanced/features/password-fields.md

```diff
@@ -0,0 +1,66 @@
+---
+id: password-fields
+title: Password fields
+---
+
+
```

Comment:
nitpick: when first benchmarking good practices for documentation, we've found that it was a good one to start with context about what readers will find on the page

For this reason, we've tried to always have a short introduction on every pages. Here's a suggestion for this page:

```suggestion
Your application will require customers to enter their password on several pages (registration, login, password forgotten…). Front-Commerce contains some default components and mechanisms for providing a good user experience while ensuring passwords match the security criteria you may have.

This page contains all information about how to adapt the default Front-Commerce behaviors.
```